admin_guide/manage_scc.adoc

```diff
@@ -11,9 +11,10 @@
 toc::[]
 
 == Overview
-You can manage
+Security context constraints allow administrators to control permissions for pods.  To learn
+more about this API type please refer to the 
 link:../architecture/additional_concepts/authorization.html#security-context-constraints[security
-context constraints] (SCCs) in your instance as normal API
+context constraints] (SCCs) API documentation.  You may manage SCCs in your instance as normal API
```

Comment:
@pweil- Did you mean "architecture documentation" here (instead of API documentation)?